packages/l/libcurl/xmake.lua

```diff
@@ -32,9 +32,9 @@ package("libcurl")
 
     -- we init all configurations in on_load, because package("curl") need it.
     on_load(function (package)
-        if package:is_plat("linux", "android", "cross") then
-            package:config_set("openssl", true)
-        end
+        -- if package:is_plat("linux", "android", "cross") then
+        --     package:config_set("openssl", true)
+        -- end
```

Comment:
why?

Reply:
When using like `add_requires("libcurl", {system = false, configs = {shared = true, mbedtls = true , openssl = false }})` on linux-arm-cross, it still using openssl, not mbedtsl as expected.

Reply:
your platform is cross, why disable it for linux/android? it will break other platforms.

you can check mbedtls configuration.

```lua
        if not package:config("mbedtls") and package:is_plat("linux", "android", "cross") then
            package:config_set("openssl", true)
        end
```

Reply:
I've tested on MSVC,  Android NDK R26D, linux gcc-x64, linux gcc-arm32.  all these works fine.  build and run and install works good.

I'm using libcurl + mbedtls both on android and arm-linux project.  
I've inspect libcurl.so + mbedtls.so with readelf -d to check things. All things is as expected.

Reply:
if you want to use mbedtls, just add `if not package:config("mbedtls") and ` to fix it.